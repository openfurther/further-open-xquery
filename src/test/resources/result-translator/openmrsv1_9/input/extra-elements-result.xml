<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!--

    Copyright (C) [2013] [The FURTHeR Project]

    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at

            http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.

-->
<ResultList>
	<!--  Person xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
		<personId>6</personId>
		<birthdate>1962-04-21</birthdate>
		<junk>blah</junk>
		<birthdateEstimated>0</birthdateEstimated>
		<causeOfDeath>0</causeOfDeath>
		<creator>1</creator>
		<dateCreated>2006-01-18</dateCreated>
		<dead>0</dead>
		<gender>M</gender>
		<sweet>awesome</sweet>
		<uuid>dd55352f-1691-11df-97a5-7038c432aabf</uuid>
		<voided>0</voided>
		<birthYear>1962</birthYear>
		<personAttributes>
			<personAttribute>
				<personAttributeId>15865</personAttributeId>
				<creator>1</creator>
				<uuid>388a4129-4f49-4fe6-8e40-32fe83b60c73</uuid>
				<value>Asian or Pacific islander</value>
				<voided>0</voided>
				<sweet>awesome</sweet>
				<personAttributeType>1</personAttributeType>
			</personAttribute>
			<personAttribute>
				<personAttributeId>21151</personAttributeId>
				<creator>1</creator>
				<uuid>ddb35c80-a14e-4fbe-8508-10f343f642c0</uuid>
				<value>Buddhist</value>
				<voided>0</voided>
				<personAttributeType>11</personAttributeType>
			</personAttribute>
			<personAttribute>
				<personAttributeId>10579</personAttributeId>
				<creator>1</creator>
				<uuid>7e22b3cd-9906-4dc6-aa42-a090c881fd69</uuid>
				<value>French</value>
				<voided>0</voided>
				<personAttributeType>9</personAttributeType>
			</personAttribute>
			<personAttribute>
				<personAttributeId>5293</personAttributeId>
				<creator>1</creator>
				<uuid>6c11f477-6730-4d4c-b09d-e18915348570</uuid>
				<value>NOT MARRIED</value>
				<junk>blah</junk>
				<voided>0</voided>
				<personAttributeType>5</personAttributeType>
			</personAttribute>
		</personAttributes>
	</Person -->
</ResultList>