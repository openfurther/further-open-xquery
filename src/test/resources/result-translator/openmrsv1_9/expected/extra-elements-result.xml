<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ResultList>
	<Person>
		<id>
			<datasetId>1234</datasetId>
			<id>0</id>
		</id>
		<compositeId>1234:0</compositeId>
		<administrativeGenderNamespaceId>30</administrativeGenderNamespaceId>
		<administrativeGender>248153007</administrativeGender>
		<dateOfBirth>1962-04-21</dateOfBirth>
		<birthYear>1962</birthYear>
		<vitalStatusNamespaceId>30</vitalStatusNamespaceId>
		<vitalStatus>371827001</vitalStatus>
		<causeOfDeathNamespaceId>30</causeOfDeathNamespaceId>
		<causeOfDeath>371827001</causeOfDeath>
	</Person>
</ResultList>